src/sagemaker_containers/transformers.py

```diff
@@ -85,7 +90,7 @@ def input_fn(input_data, content_type):
         Returns:
             (obj): data ready for prediction.
         """
-        return encoder.DefaultDecoder().decode(input_data, content_type)
+        return encoders.default_decoder.decode(input_data, content_type)
```

Comment:
make self.decoder attr in this class's __init__ method instead of using module var in encoders

Reply:
discussed offline. ok as is.